### ISSUE TITLE
Seed with softlock

### ISSUE THREAD

**Issue by patrickbila** [IntelOrca/biorand#748]
### Version

3.1.8 (2f583cd)

### Seed

R730-Z440-RCHH4HQ5Z9Y0002049K496ZGZZZZZZZZZZZZZZZZZZZZZZZZZ

### Scenario

RE 3 (Jill)

### Player Character

Jill

### What happened?

2 Carlos' itens appeared in clock tower. The Tape Recorder appeared on Clock Tower courtyard (impossible to access on Carlos segment). The Vacine Base appeared on Clock puzzle room (maybe not a issue cause is accessable on Carlos segment), but I don't know if Jill' segments itens keep here when playing as Carlos. 

1 Jill's item appeared on hospital. The Iron Pipe appeared on 1F Hospital save room (impossible to acess on Jill segment). I tried not to get both original vaccine's parts (that triggers Hospital explosion) but with the Vaccine complete (As base vacine had been got with Jill I put it with debug). But trying to use the complete vacine on Jill without triggering the explosion, the Vaccine does't work.

So Carlos' key items can't appear on Jill segmente, and vice versa. Also, magnum appeared on Carlos segmente (on this seed and in another. Also on Carlos segmente appeared gunpowders, Jill's weapons bullets. Even its not an issue, it would be better if on Carlos segment only appear items that Carlos can use (due Jill can't acess hospital later).

**Comment by IntelOrca** [comment]
Looks like this is caused by disabling segmented placement. The Carlos segment is an edge case the item randomizer can't currently cope with very well.

**Comment by patrickbila** [comment]
@IntelOrca yes is was without segmented placement. but you guys could fix it whether you play enable or not. If enable, shoud ignore Carlos segment, and in his segment shoud only appear his key (sickroom key, vaccine parts) itens, and only itens used by him (MG ammo, pistol ammo, herbs, sprays, inks)

**Comment by IntelOrca** [comment]
> @IntelOrca yes is was without segmented placement. but you guys could fix it whether you play enable or not. If enable, shoud ignore Carlos segment, and in his segment shoud only appear his key (sickroom key, vaccine parts) itens, and only itens used by him (MG ammo, pistol ammo, herbs, sprays, inks)

Ignoring isn't that simple to do. Just design constraints with the item randomizer. I will look at fixing this at some point.
